src/content/developers/tutorials/run-node-raspberry-pi/index.md

```diff
@@ -1,264 +1,153 @@
-
-Basically, both are the same image and include the same features of the Raspbian based images. But they are setup for running execution layer or consensus layer software by default.
-
-**Images take care of all the necessary steps**, from setting up the environment and formatting the SSD disk to installing and running the Ethereum software as well as starting the blockchain synchronization.
-
-## Main features {#main-features}
-
-- Based on Ubuntu 20.04 64bit
-- Automatic USB disk partitioning and formatting
-- Adds swap memory (ZRAM kernel module + a swap file) based on Armbian work [[7]](/developers/tutorials/run-node-raspberry-pi/#references)
-- Changes the hostname to something like “ethnode-e2a3e6fe” based on MAC hash
-- Runs software as a systemd service and starts syncing the Blockchain
-- Includes an APT repository for installing and upgrading Ethereum software
-- Includes a monitoring dashboard based on Grafana / Prometheus
-
-## Software included {#software-included}
-
-Both images include the same packages, the only difference between them is that the execution version runs Geth by default and the consensus version runs Prysm beacon chain by default.
-
-### Execution clients {#execution-clients}
-
-- Geth [[8]](/developers/tutorials/run-node-raspberry-pi/#references): 1.9.13 (official binary)
-- Parity [[9]](/developers/tutorials/run-node-raspberry-pi/#references): 2.7.2 (cross compiled)
-- Nethermind [[10]](/developers/tutorials/run-node-raspberry-pi/#references): 1.8.28 (cross compiled)
-- Hyperledger Besu [[11]](/developers/tutorials/run-node-raspberry-pi/#references): 1.4.4 (compiled)
-
-### Consensus clients {#consensus-clients}
-
-- Prysm [[12]](/developers/tutorials/run-node-raspberry-pi/#references): 1.0.0-alpha6 (official binary)
-- Lighthouse [[13]](/developers/tutorials/run-node-raspberry-pi/#references): 0.1.1 (compiled)
-
-### Ethereum framework {#ethereum-framework}
-
-- Swarm [[14]](/developers/tutorials/run-node-raspberry-pi/#references): 0.5.7 (official binary)
-- Raiden Network [[15]](/developers/tutorials/run-node-raspberry-pi/#references): 0.200.0~rc1 (official binary)
-- IPFS [[16]](/developers/tutorials/run-node-raspberry-pi/#references): 0.5.0 (official binary)
-- Statusd [[17]](/developers/tutorials/run-node-raspberry-pi/#references): 0.52.3 (compiled)
-- Vipnode [[18]](/developers/tutorials/run-node-raspberry-pi/#references): 2.3.3 (official binary)
-
-## Installation guide and usage {#installation-guide-and-usage}
-
-### Recommended hardware and setup {#recommended-hardware-and-setup}
-
-- Raspberry 4 (model B) - 4GB
+- Raspberry 4 (model B 8GB)
 - MicroSD Card (16 GB Class 10 minimum)
-- SSD USB 3.0 disk (see storage section)
+- 1 TB SSD minimum USB 3.0 disk or an SSD with an USB to SATA case (see Storage section).
```

Comment:
Would recommend 2 TB min right now unfortunately... Geth already runs ~1 TB alone.
Also, the `#storage` header was removed, so there is no "Storage" section =)

```suggestion
- 2 TB SSD minimum USB 3.0 disk or an SSD with a USB to SATA case.
```

Reply:
+1